typescript/cli/src/deploy/core.ts

```diff
@@ -29,18 +30,15 @@ interface DeployParams {
   chain: ChainName;
   config: CoreConfig;
 }
+
+interface ApplyParams extends DeployParams {
+  mailbox: Address;
+}
 /**
  * Executes the core deploy command.
  */
-export async function runCoreDeploy({
-  context,
-  chain,
-  config,
-}: {
-  context: WriteCommandContext;
-  chain: ChainName;
-  config: CoreConfig;
-}) {
+export async function runCoreDeploy(params: DeployParams) {
+  let { context, chain, config } = params;
```

Comment:
Updated to use DeployParams. using `let` instead of `const` because chain gets updated due to dry run

Reply:
in that case
```suggestion
  const { context, config } = params;
  let chain = params.chain;
```
to be explicit?

Reply:
this doesn't actually pass prettier anyway

Reply:
fixed